CRT.c

```diff
@@ -575,9 +578,11 @@ void CRT_restorePrivileges() {
 
 #endif /* HAVE_SETUID_ENABLED */
 
-// TODO: pass an instance of Settings instead.
+static void *backtraceArray[256];
```

Comment:
Can't this variable be declared in `CRT_handleSIGSEGV()` (without `static`) ?

Reply:
Potentially yes. I'll have a look, when I'm back near my notebook.